<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPage</class>
 <widget class="QMainWindow" name="AdminPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>896</width>
    <height>649</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <property name="spacing">
     <number>11</number>
    </property>
    <property name="leftMargin">
     <number>18</number>
    </property>
    <property name="topMargin">
     <number>18</number>
    </property>
    <property name="rightMargin">
     <number>19</number>
    </property>
    <property name="bottomMargin">
     <number>18</number>
    </property>
    <item alignment="Qt::AlignHCenter">
     <widget class="QLabel" name="label">
      <property name="sizePolicy">
       <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
        <horstretch>0</horstretch>
        <verstretch>0</verstretch>
       </sizepolicy>
      </property>
      <property name="font">
       <font>
        <family>幼圆</family>
        <pointsize>28</pointsize>
       </font>
      </property>
      <property name="text">
       <string>订票系统后台</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignHCenter|Qt::AlignTop</set>
      </property>
     </widget>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <property name="spacing">
       <number>60</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="spacing">
         <number>80</number>
        </property>
        <property name="topMargin">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="flight_info_button">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>航班信息</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="money_ratio_button">
          <property name="font">
           <font>
            <family>幼圆</family>
            <pointsize>18</pointsize>
           </font>
          </property>
          <property name="text">
           <string>收入和上座率</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Expanding</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <property name="spacing">
         <number>47</number>
        </property>
        <item>
         <widget class="QStackedWidget" name="stackedWidget">
          <property name="currentIndex">
           <number>1</number>
          </property>
          <widget class="QWidget" name="page">
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="spacing">
             <number>9</number>
            </property>
            <property name="leftMargin">
             <number>7</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="rightMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_11">
              <item>
               <widget class="QRadioButton" name="change_button">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>修改</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="add_button">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>增加</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_4">
              <property name="spacing">
               <number>0</number>
              </property>
              <property name="sizeConstraint">
               <enum>QLayout::SetDefaultConstraint</enum>
              </property>
              <property name="topMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="fid_label">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>fid：</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QLineEdit" name="fid_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_2">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_2">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>航班类型：</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QLineEdit" name="ftype_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_12">
              <item>
               <widget class="QLabel" name="label_3">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>出发地：</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QLineEdit" name="from_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_5">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_5">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>目的地：</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QLineEdit" name="to_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>出发时间：</string>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QDateTimeEdit" name="begin_timeedit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_7">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_7">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>到达时间：</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QDateTimeEdit" name="end_timeedit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_8">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_8">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>餐食：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="food_yes_radio">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>是</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QRadioButton" name="food_no_radio">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>否</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_9">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_9">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>座位数：</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QLineEdit" name="sitenum_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_10">
              <property name="topMargin">
               <number>0</number>
              </property>
              <item alignment="Qt::AlignLeft">
               <widget class="QLabel" name="label_10">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>收入：</string>
                </property>
                <property name="scaledContents">
                 <bool>true</bool>
                </property>
                <property name="alignment">
                 <set>Qt::AlignCenter</set>
                </property>
               </widget>
              </item>
              <item alignment="Qt::AlignLeft">
               <widget class="QLineEdit" name="income_edit">
                <property name="sizePolicy">
                 <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                  <horstretch>0</horstretch>
                  <verstretch>0</verstretch>
                 </sizepolicy>
                </property>
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="alignment">
                 <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item alignment="Qt::AlignHCenter">
             <widget class="QPushButton" name="ensure_button">
              <property name="sizePolicy">
               <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
                <horstretch>0</horstretch>
                <verstretch>0</verstretch>
               </sizepolicy>
              </property>
              <property name="font">
               <font>
                <family>幼圆</family>
                <pointsize>18</pointsize>
               </font>
              </property>
              <property name="text">
               <string>确定</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="page_2">
           <layout class="QVBoxLayout" name="verticalLayout">
            <property name="spacing">
             <number>23</number>
            </property>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_13">
              <item>
               <widget class="QLabel" name="money_fid_label">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>输入航班fid：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLineEdit" name="money_fid_edit">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="pushButton">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>查询</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_14">
              <item>
               <widget class="QLabel" name="label_11">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>上座率：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="sitenum_label">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>sitenum</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="label_13">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>/</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="totalnum_label">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>totalnum</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_3">
              <property name="bottomMargin">
               <number>0</number>
              </property>
              <item>
               <widget class="QLabel" name="label_15">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>本次航班收入金额：</string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QLabel" name="money_label">
                <property name="font">
                 <font>
                  <family>幼圆</family>
                  <pointsize>18</pointsize>
                 </font>
                </property>
                <property name="text">
                 <string>money</string>
                </property>
               </widget>
              </item>
             </layout>
            </item>
            <item>
             <spacer name="verticalSpacer_2">
              <property name="orientation">
               <enum>Qt::Vertical</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>20</width>
                <height>40</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>896</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
